src/components/v2/AmountForm/validateForm.ts

```diff
@@ -0,0 +1,15 @@
+import { FormValues } from './types';
+
+const validateForm = (values: FormValues) => {
+  const errors: {
+    amount?: string;
+  } = {};
+
+  if (!values.amount || values.amount.isEqualTo(0)) {
+    errors.amount = 'required';
```

Comment:
I'd like to use `yup` for validation. What are your thoughts?

Reply:
I wanted to originally, but in this case I feel it's adding more complexity than anything, because we'd have to create a custom yup type to and `BigNumber` values, which means also having to extend yup's Typescript definition in order to support our custom type. This feels like a lot of overhead for a use case that's only relevant to this specific form.

As far as I can see, the only other type of form we have in the app is the form for proposing a vote. For this one I think using `yup` would make sense since the values used will be supported out of the box.